cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -397,7 +397,7 @@ runTransactionBuildEstimateCmd -- TODO change type
     , txBodyOutFile
     } = do
     let sbe = convert currentEra
-        meo = convert currentEra
+        meo = convert (convert currentEra :: BabbageEraOnwards era)
```

Comment:
Why not also implement:
```
instance Convert Era MaryEraOnwards where
```

We can avoid calling `convert` twice.

Reply:
That was the original issue, wasn't it?

Reply:
Yes this is what we want to avoid indeed :smile: